lms/djangoapps/program_enrollments/admin.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+"""
+Admin tool for the Program Enrollments models
+"""
+from __future__ import unicode_literals
+
+# from django.contrib import admin
+
+# Register your models here.
```

Comment:
You should register the program enrollment mode for admin here.

Reply:
Should I just register it, or do you think it will be worth having the admin tool fully baked out for this model?

Reply:
Just register it should be fine for now